core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -181,25 +207,23 @@ static paddr_t core_mmu_get_ul1_ttb_pa(void)
 {
 	paddr_t pa = virt_to_phys((void *)core_mmu_get_ul1_ttb_va());
 
-	if (pa & ~TEE_MMU_TTB_UL1_MASK)
+	if (pa & ~TTB_UL1_MASK)
 		panic("invalid user l1 table");
 	return pa;
 }
 
-static void *core_mmu_alloc_l2(struct tee_mmap_region *mm)
+static void *core_mmu_alloc_l2(size_t size)
```

Comment:
This could could be aligned on core_mu_lpae.c implementation:
```
 		uint32_t *new_table = main_mmu_l2_ttb[next_xlat++];
		if (next_xlat > MAX_XLAT_TABLES)
			panic("running out of xlat tables");
```